CHANGELOG.rst

```diff
@@ -1,3 +1,10 @@
+* 1.3.5, 6th August 2015
+
+  - Move gzipstream module to smart_open package
+  - Make sure reader objects never return None
+  - Fix issue #85
```

Comment:
Too cryptic -- describe the actual change.

Also, we can hyperlink to the actual PR / more information, where appropriate (not just text).